spec/support/solidus_stripe/webhook/event_with_context_factory.rb

```diff
@@ -56,6 +57,11 @@ def stripe_object
         @stripe_object ||= Stripe::Event.construct_from(data)
       end
 
+      def solidus_stripe_object
```

Comment:
Can you please help me understand why this method is called `solidus_stripe_object` and returns a Webhook Event?

Reply:
The `EventWithContext` class can return a Stripe event in different representations:

- `#json`: A JSON string representation
- `#signature_header`: How it looks signed in the request header
- `#stripe_object`: As a `Stripe::Event` object.
- `#solidus_stripe_object`: As a `SolidusStripe::Webhook::Event` object.

I know it's a bit cryptic, but I can't find better candidates. We can't call the `#stripe_object` method `#stripe_event`, because all those methods return representations of a Stripe event. Once we have that method, the most logical candidate for the new addition is `#solidus_stripe_object`.

Happy to change it if we find a better solution!!